packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.spec.ts

```diff
@@ -825,6 +825,151 @@ describe("DefaultVisualizers unit tests", () => {
 		expect(result).to.deep.equal(expected);
 	});
 
+	it("SharedTree: Leaf Handle", async () => {
```

Comment:
I think this is a bit more descriptive of the scenario for the test? Because handles are always leaves of the SharedTree, like in the next test. What's special about this one is that a handle is the only thing in the tree.

```suggestion
	it("SharedTree: Handle at the root", async () => {
```

Reply:
I guess you're right. At least in the visualizer context, we are only interested in the `handle` in the `leaf` node side. 

I was trying to imply that there only exists a single element (`handle`) in the tree.